Feature/issue 183

riccardogenova opened the pull request:
I'm working on function/logic to convert url to correct path and before push the last commit I do a code review.

storrisi commented on src/components/PageDefinitions.js:
```diff
@@ -256,11 +256,33 @@ class PageDefinitions extends AbstractPageDefinitions {
+      this.onTextDrop(text)
+    } else {
+      const files = Object.values(e.dataTransfer.files)
+      const acceptedFiles = files[0].type === 'application/json'
```
What about if files are multiple and some of them are not in the accepted format?
Another thing: `=== 'application/json'` couldn't be the only check that we will need to do.

I suggest to implement some methods in a utils/ class in order to pass dropped elements (files or texts) and do any check inside it.

Files checks could be multiple and not only `application/json`, and if a list of files contains one ore more rejected extensions , it shouldn't fail entirely, but otherwise parse only the accepted files (this is something is already happening i guess, check the codebase)

riccardogenova commented:
I created some utils function for path formatting (all providers available) and moved all of them to the EntitySpec class component. 

I deleted the remote branch because there were some conflicts during the rebase so I preferred solved them locally and push branch remotely back.
